src/main/java/org/gridsuite/study/server/NetworkModificationService.java

```diff
@@ -398,19 +398,30 @@ public void stopBuild(@NonNull UUID studyUuid, @NonNull UUID nodeUuid) {
         restTemplate.put(getNetworkModificationServerURI(false) + path, null);
     }
 
-    public void reorderModification(UUID groupUuid, UUID modificationUuid, UUID beforeUuid) {
+    private HttpEntity<String> getModificationsUuidBody(List<UUID> modificationUuidList) {
+        HttpEntity<String> httpEntity;
+        try {
+            HttpHeaders headers = new HttpHeaders();
+            headers.setContentType(MediaType.APPLICATION_JSON);
+            httpEntity = new HttpEntity<>(objectMapper.writeValueAsString(modificationUuidList), headers);
+        } catch (JsonProcessingException e) {
+            throw new UncheckedIOException(e);
+        }
+        return httpEntity;
+    }
+
+    public void reorderModification(UUID groupUuid, List<UUID> modificationUuidList, UUID beforeUuid) {
         Objects.requireNonNull(groupUuid);
-        Objects.requireNonNull(modificationUuid);
-        var path = UriComponentsBuilder.fromPath(GROUP_PATH
-                + DELIMITER + MODIFICATIONS_PATH + DELIMITER + "move")
-            .queryParam("modificationsToMove", modificationUuid);
+        var path = UriComponentsBuilder.fromPath(GROUP_PATH)
+            .queryParam("action", "MOVE");
```

Comment:
Why not implement GroupModificationAction within the study server too ?

Reply:
Good point. I will decide with @SlimaneAmar if we change them.

Reply:
We wont change study server cause the logic is a bit different